packages/sanity/test/__snapshots__/exports.test.ts.snap

```diff
@@ -18,7 +18,6 @@ exports[`exports snapshot 1`] = `
     "ArrayOfPrimitivesItem": "function",
     "AutoCollapseMenu": "object",
     "AvatarSkeleton": "object",
-    "BasicDocument": "function",
```

Comment:
Is this consumed externally?

Reply:
nope, this was added recently so safe to move